ResourcesCard.jsx

```diff
@@ -1 +1,83 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = (resourceUuid) => {
+    const evt = new Object();
+    evt = {
+      event_name: "web_dashboard_card_" + r.id + "_" + r.title + "_tap"
+    };
+    sendEvt(evt);
+    console.log("sending event");
```

Comment:
We can shorten the lines of code here by doing - 

```suggestion
    const evt = {
          event_name: "web_dashboard_card_" + r.id + "_" + r.title + "_tap"
    }
    sendEvt(evt);
    console.log("sending event");
```

Reply:
Line 16: would prefer us to use string interpolation for easier reading.

Reply:
Line 19: We usually dont have `console.log` in production. Did something break in our linter?